Makefile.am

```diff
@@ -269,7 +268,7 @@ install-exec-hook:
 			cd ${DESTDIR}${bindir} && ln -f -s @MPIFORT_NAME@ @MPIF90_NAME@ ; \
 		fi ; \
 	fi ; \
-	if test -e ${DESTDIR}${bindir}/@MPIFORT_NAME@ -a ! -e ${DESTDIR}${bindir}/@MPIF77_NAME@ ; then \
```

Comment:
I sneak in this consistency change. Is it correct or am I missing something?

Reply:
Looks good.